xml/System.Data/DataRowComparer`1.xml

```diff
@@ -181,11 +186,13 @@
         <Parameter Name="rightRow" Type="TRow" />
       </Parameters>
       <Docs>
-        <param name="leftRow">The first <see cref="T:System.Data.DataRow" /> object to compare.</param>
-        <param name="rightRow">The second <see cref="T:System.Data.DataRow" /> object to compare.</param>
-        <summary>Compares two <see cref="T:System.Data.DataRow" /> objects by using a column-by-column, value-based comparison.</summary>
-        <returns>
-          <see langword="true" /> if the two <see cref="T:System.Data.DataRow" /> objects have ordered sets of column values that are equal; otherwise, <see langword="false" />.</returns>
+        <param name="leftRow">The first input DataRow</param>
```

Comment:
Bad docs making their way in, presumably from https://nuget.info/packages/System.Data.DataSetExtensions.